src/GitHook/Command/FileCommand/PreCommit/StyleLintCommand.php

```diff
@@ -42,7 +42,7 @@ public function run(CommandContextInterface $context): CommandResultInterface
     {
         $commandResult = new CommandResult();
 
-        $processDefinition = ['npx', 'stylelint', $context->getFile(), '--config', 'node_modules/@spryker/frontend-config.stylelint/.stylelintrc', '--fix'];
+        $processDefinition = ['node', './frontend/libs/stylelint', 'NODE_ENV=development', 'development', 'fix', $context->getFile()];
```

Comment:
Shouldn't `NODE_ENV=development` be in the beginning of the command?

Reply:
when I placed it in the 1st position git-hooks didn't work.

Reply:
updated arguments